src/app/personside/infotabs/meldinger/traadliste/TraadListeElement.tsx

```diff
@@ -1,25 +1,48 @@
 import * as React from 'react';
-import { Traad } from '../../../../../models/meldinger/meldinger';
+import { Meldingstype, Traad } from '../../../../../models/meldinger/meldinger';
 import VisMerKnapp from '../../../../../components/VisMerKnapp';
 import Element from 'nav-frontend-typografi/lib/element';
 import styled from 'styled-components';
 import { meldingstypeTekst, temagruppeTekst } from '../utils/meldingstekster';
 import Normaltekst from 'nav-frontend-typografi/lib/normaltekst';
 import { theme } from '../../../../../styles/personOversiktTheme';
-import SakIkkeTilgangIkon from '../../../../../svg/SakIkkeTilgangIkon';
 import { formatterDatoTid } from '../../../../../utils/dateUtils';
-import { sisteSendteMelding } from '../utils/meldingerUtils';
+import { erMonolog, sisteSendteMelding } from '../utils/meldingerUtils';
+import OppmoteIkon from '../../../../../svg/OppmoteIkon';
+import TelefonIkon from '../../../../../svg/TelefonIkon';
+import OppgaveIkon from '../../../../../svg/OppgaveIkon';
+import DokumentIkon from '../../../../../svg/DokumentIkon';
+import MonologIkon from '../../../../../svg/MonologIkon';
+import DialogIkon from '../../../../../svg/DialogIkon';
+import { AppState } from '../../../../../redux/reducers';
+import { connect } from 'react-redux';
+import { AsyncDispatch } from '../../../../../redux/ThunkTypes';
+import { settValgtTraad } from '../../../../../redux/meldinger/actions';
 
-interface Props {
+interface OwnProps {
     traad: Traad;
-    erValgtTraad: boolean;
+}
+
+interface StateProps {
+    valgtTraad?: Traad;
```

Comment:
for å øke performance kan du gjøre dette i selectoren din istedenfor, da slipper komponenten å rendres på nytt hver gang det velges en ny tråd.
```
interface StateProps {
  erValgtTraad: boolean;
}
```